util/workers.py

```diff
@@ -28,6 +28,12 @@ def get_worker_count(worker_kind_name, multiplier, minimum=None, maximum=None):
     minimum = minimum or multiplier
     maximum = maximum or (multiplier * multiplier)
 
+    minimum_override = os.environ.get("WORKER_COUNT_UNSUPPORTED_MINIMUM")
```

Comment:
@kleesc what do you think? We need only 1 worker for development, and I want to make our development env closer to production.

Reply:
Worker count is based on the number of cores running. Shouldn't need to change this. The number of gunicorn workers running in the background should not matter for a high level, when developing on Quay

Reply:
There are two problems:

1. resource consumption by the local dev env
2. lack of the common SECRET_KEY in `local-dev`

By scaling down everything to a single replica I've got the same environment as the old `local-dev`, i.e. no predefined `SECRET_KEY` and only one replica with in-memory secret. If we are ok with increased resource consumption, we can add a static SECRET_KEY to the local-dev config like we did with DATABASE_SECRET_KEY.